docs(main): introduction

eToThePiIPower opened the pull request:
Introduce the developer to the dojo
- name
- occupation

Create the introduction file to introduce the developer to the dojo.

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

eToThePiIPower committed:
docs(main): introduction

Introduce the developer to the dojo
- name
- occupation

Create the introduction file to introduce the developer to the dojo.

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1,2 @@
+# Introduction
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

eToThePiIPower committed:
docs(main): add codeschool path to introduction

Add the developer's favorite code school path to the introduction

eToThePiIPower commented:
Done! Favorite code school path has been added.

eToThePiIPower committed:
Initial kill list

eToThePiIPower committed:
add initial deadly skills

eToThePiIPower committed:
add killing history skill

codeschool-kiddo committed:
Added new rule about coffee

eToThePiIPower committed:
fix typos

eToThePiIPower committed:
Merge pull request #3 from eToThePiIPower/new_rules

Too many people are spilling coffee

eToThePiIPower committed:
Merge branch 'deadly_skills'

eToThePiIPower committed:
Replace assassins with bad code practices

We wanted to replace assassins with more worthwhile targets, so we're killing
bad coding practices instead

eToThePiIPower committed:
Merge tag 'v1.0.1'

Replace assassins with bad coding practices

Conflicts:
	kill_list.md

eToThePiIPower committed:
fixes #4

eToThePiIPower committed:
Merge pull request #5 from eToThePiIPower/kill_list

fixes #4

eToThePiIPower committed:
Expand README

* Add reminder to read the rules
* Add link to original organization

eToThePiIPower committed:
Merge branch 'master' of github:eToThePiIPower/dojo_rules
